src/utils/getS3SignedUrl.js

```diff
@@ -0,0 +1,26 @@
+const AWS = require('aws-sdk');
+const config = require('../config');
+
+const getS3SignedUrl = (operation, params) => {
+  const s3 = new AWS.S3({
+    apiVersion: '2006-03-01',
+    signatureVersion: 'v4',
+    region: config.awsRegion,
+  });
+
+  const defaultParams = {
+    Expires: 120, // seconds; default to 2 mins
```

Comment:
we are talking about s3 operations that could take more than 2 minutes, I think default should be the aws default (15 minutes I believe? maybe check that, or just not pass expire and leave that default, whatever you feel more comfortable with).

Reply:
Hmm... ok, let's just use the default AWS settings . Using the this `defaultParams` is also not good as it forces the user to know about this settings.